healthz/checks/checks.go

```diff
@@ -0,0 +1,59 @@
+package checks
```

Comment:
Note - I think linter should check this, you can add license header check to test  like here https://github.com/kubernetes/helm/blob/master/scripts/validate-license.sh